### ISSUE TITLE
Update a patient's info

### ISSUE THREAD

**Issue by abbyad** [medic/medic-webapp#842]
Right now a patient's info cannot be updated after the initial registration. A "visit report" can be sent to mute a group of scheduled messages, but the LMP, or name cannot be updated. This type of update will be increasingly important, especially as we build more patient-centric features in the next few months.

This can have compounding implications, because updating the LMP could mean rescheduling all of the messages. Further discussion is needed on how we would go about this.

**Comment by garethbowen** [comment]
I think this is a duplicate of #557

**Comment by abbyad** [comment]
I should have been clearer... I had meant "Updating a patient's info with a report", which is different than editing a submitted report. 

Functionally this is the difference between a CHW sending a follow up report on a patient that completes/changes previous info versus a manager using the webapp to update an incorrect value in a stock report.

**Comment by SCdF** [comment]
@alxndrsn please close or schedule before the end of this sprint. See [triaging old issues](https://github.com/medic/medic-docs/blob/master/md/dev/workflow.md#triaging-old-issues).

**Comment by sglangevin** [comment]
Once https://github.com/medic/medic-webapp/issues/3074 is done then this will be possible for person-centered workflows.

**Comment by alxndrsn** [comment]
This doesn't seem to be of any priority.  @sglangevin please re-open if this needs scheduling

**Comment by abbyad** [comment]
This feature request is relevant still, perhaps moreso, so that we can update the person's name, or age, or any of the fields on the person's profile. I'll let @sglangevin prioritize when needed, or open a new issue as needed.

**Comment by sglangevin** [comment]
Let's open a new issue once we have a use case for this. For now, we will have the ability to clear a schedule with a new registration, which should be enough to accomplish what is currently needed.
